src/languages/xml.js

```diff
@@ -6,10 +6,27 @@ Category: common
 
 function(hljs) {
   var XML_IDENT_RE = '[A-Za-z0-9\\._:-]+';
-    var XML_ENTITIES = {
+  var XML_ENTITIES = {
     className: 'symbol',
     begin: '&[a-z]+;|&#[0-9]+;|&#x[a-f0-9]+;'
   };
+  var XML_META_KEYWORDS = {
+	  className: 'meta-keyword',
+	  begin: '\\s#?[a-z_][a-z1-9_-]+',
+	  illegal: '\\n',
+  };
+  var XML_META_PAR_KEYWORDS = {
+	  begin: '\\(', end: '\\)',
+	  contains:[
+	    {
+	      className: 'meta-keyword',
+	      begin: '#?[a-z_][a-z1-9_-]+',
```

Comment:
Looks like 15/23 share a common constant that you could factor out.

Why does Meta eat a space though but not meta_par?

Reply:
Outside the parentheses, spaces define the keywords. No space is required within parentheses. I came up with another solution that also addresses the following criticism.

Reply:
Generally I'd dislike something like line 14 because it's too broad (it matches ANY spaces anywhere). BUT since this is already scoped inside of ONLY `meta` modes then this is ok, since that limits the scope plenty.